app/assets/javascripts/services/audio_player.js

```diff
@@ -1,7 +1,12 @@
 belAir.service('AudioPlayer', ['$http', function AudioPlayerService($http) {
   var audio = this;
-  var player = new MediaElementPlayer('#audio-player', {});
-  var voscastError = "Failed to load because no supported source was found.";
```

Comment:
Just realized this is the thread for that PR.

On Tue, May 10, 2016 at 12:52 AM Steve Ellis <steve@steveell.is> wrote:

> It's a little different in my recent pull request, we at least don't save
> the expected error text(stopped using it in a past commit)... But the error
> is still there.
>
> I told this to Mike in person, to document it:
> The error itself is tough to catch because it's actually raised by native
> browser code when the audio tag fails to play its source. We handle it by
> attaching an 'onerror' callback which picks a random episode to play if the
> player fails. Pretty much we let the audio player detect if the live stream
> is playing or not. Downside is you still get a little error in the console
> :(
>
> If you can think of a good way to catch that, I'm open to it. It might not
> be hard, I don't know. What I do know: when manually calling play on the
> audio tag, Chrome returns a promise(catchable), FF does not.
> On Tue, May 10, 2016 at 12:37 AM Karl Hohn <notifications@github.com>
> wrote:
>
>> In app/assets/javascripts/services/audio_player.js
>> <https://github.com/mikekaminsky/belair/pull/28#discussion_r62611213>:
>>
>> > @@ -1,7 +1,12 @@
>> >  belAir.service('AudioPlayer', ['$http', function AudioPlayerService($http) {
>> >    var audio = this;
>> > -  var player = new MediaElementPlayer('#audio-player', {});
>> > -  var voscastError = "Failed to load because no supported source was found.";
>>
>> is this error being handled differently now?
>>
>> —
>> You are receiving this because you authored the thread.
>> Reply to this email directly or view it on GitHub
>> <https://github.com/mikekaminsky/belair/pull/28/files/dc8d543c2a42be74b8145e3e2e692193aef7a9cb#r62611213>
>>
>